scripts/sbom-python-node.sh

```diff
@@ -15,9 +15,9 @@ BOMDIR=$(mktemp -d)
 # Generate the frontend ui SBOM
 # The primary reason for using cdxgen (and then converting to spdx format via syft)
 # is that it can look up nodeJS license fields online
-FETCH_LICENSE=true gum spin --spinner dot --title \
-  "Generating CycloneDX SBOM for the Frontend UI packages. Fetching the licenses from online may take a while..." \
-  -- cdxgen plugins/web/ui -o $BOMDIR/ui_cyclonedx.json
+# FETCH_LICENSE=true gum spin --spinner dot --title \
+#   "Generating CycloneDX SBOM for the Frontend UI packages. Fetching the licenses from online may take a while..." \
+#   -- cdxgen plugins/web/ui -o $BOMDIR/ui_cyclonedx.json
```

Comment:
Might as well remove the commented-out lines. (and probably rename the script)

Reply:
Done.